Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiManager.cpp

```diff
@@ -152,17 +152,19 @@ namespace AzToolsFramework::ViewportUi
     {
         if (auto clusterIt = m_clusterButtonGroups.find(clusterId); clusterIt != m_clusterButtonGroups.end())
         {
+            auto elementId = clusterIt->second->GetViewportUiElementId();
             m_clusterButtonGroups.erase(clusterIt);
-            m_viewportUi->RemoveViewportUiElement(clusterIt->second->GetViewportUiElementId());
+            m_viewportUi->RemoveViewportUiElement(elementId);
```

Comment:
Nit: What about just reordering the `erase` call to come second? That element id is just a 64-bit integer, but this has to copy it.

Reply:
I will try it.  I don't understand enough about viewport UI though... to want to change too much in stabilization.

Reply:
looks like it works fine. .. updated.